prow/config/trustworthy-jwt.yaml

```diff
@@ -1,6 +1,8 @@
 # This configs KinD to spin up a k8s cluster with trustworthy jwt (Service Account Token Volume Projection) feature.
 kind: Cluster
 apiVersion: kind.sigs.k8s.io/v1alpha3
+networking:
+  ipFamily: ${IP_FAMILY:-ipv4}
```

Comment:
is kind evaluating environment variables in config files?

Reply:
ups, you are right, is a heredoc

https://github.com/kubernetes-sigs/kind/blob/master/hack/ci/e2e-k8s.sh#L117-L123

let me find another way